### ISSUE TITLE
Valeurs ebc

### ISSUE THREAD

**Issue by joebar64** [314r/joliebulle#148]
Bonjour,
En important des fichiers beerxml dans joliebulle les valeurs de l'ebc ne correspondent pas. 
Tout est divisé par deux dans joliebulle. Du coup lors de l'import ça multiplie tous les ebc par deux.
Je n'ai pas testé pour les exports.
Merci.

**Comment by 314r** [comment]
Bonjour,
Le standard Beerxml demande que les couleurs soient enregistrées en SRM. Joliebulle affiche des EBC. Donc il est normal que les valeurs ne soient pas les mêmes. Le logiciel fait une conversion à l'ouverture du fichier, et à l'enregistrement.
Est-ce que ça répond à ta question ? Concrètement, tu trouves des résultats discordants ?

**Comment by joebar64** [comment]
Merci pour la réponse. Si j'ai bien compris, la valeur SRM serait la moitié de la valeur EBC ?Concrètement, j'ai les valeurs doublées dans joliebulle. Mais le problème peut venir de l'autre soft qui exporterait en EBC si le rapport est bien de 2.Le 9 déc. 2017 22:53, 314r <notifications@github.com> a écrit :Bonjour,
Le standard Beerxml demande que les couleurs soient enregistrées en SRM. Joliebulle affiche des EBC. Donc il est normal que les valeurs ne soient pas les mêmes. Le logiciel fait une conversion à l'ouverture du fichier, et à l'enregistrement.
Est-ce que ça répond à ta question ? Concrètement, tu trouves des résultats discordants ?

**Comment by joebar64** [comment]
Désolé, après recherche c'est quasi ça. SRM = EBC × 0.508le soucis ne vient pas de joliebulle... autant pour moi.Le 10 déc. 2017 12:07, Sébastien Lapeyre <sebastien.lapeyre64@free.fr> a écrit :Merci pour la réponse. Si j'ai bien compris, la valeur SRM serait la moitié de la valeur EBC ?Concrètement, j'ai les valeurs doublées dans joliebulle. Mais le problème peut venir de l'autre soft qui exporterait en EBC si le rapport est bien de 2.Le 9 déc. 2017 22:53, 314r <notifications@github.com> a écrit :Bonjour,
Le standard Beerxml demande que les couleurs soient enregistrées en SRM. Joliebulle affiche des EBC. Donc il est normal que les valeurs ne soient pas les mêmes. Le logiciel fait une conversion à l'ouverture du fichier, et à l'enregistrement.
Est-ce que ça répond à ta question ? Concrètement, tu trouves des résultats discordants ?

**Comment by 314r** [comment]
Voilà, EBC = 1.97 * SRM.
Tu remarques des valeurs doublées : 
- si c'est par rapport aux valeurs inscrites dans le fichier, c'est donc normal.
- si c'est par rapport à un autre logiciel, il y a un problème. Dans mon souvenir, par le passé le problème s'est déjà posé avec Beersmith.

**Comment by 314r** [comment]
Nos messages se sont croisés.

**Comment by joebar64** [comment]
Salut,Au cas où j'ai trouvé ça :http://www.beerxml.com/beerxml.htmL'unité de référence serait le lovibond (ou SRM mais pour l'extrait ?) avec ces formules de calcul :https://www.brewtoad.com/tools/color-converterOn aurait EBC = ( 2.668562 × °L ) - 1.4972Seb.Le 10 déc. 2017 12:11, 314r <notifications@github.com> a écrit :Voilà, EBC = 1.97 * SRM.
Tu remarques des valeurs doublées :
si c'est par rapport aux valeurs inscrites dans le fichier, c'est donc normal.si c'est par rapport à un autre logiciel, il y a un problème. Dans mon souvenir, par le passé le problème s'est déjà posé avec Beersmith.

**Comment by 314r** [comment]
Oui, c'est vrai que pour être rigoureux le beerxml utilise le Lovibond, je n'avais plus ça en tête.
A l'époque des premiers essais de calculs pour Joliebulle la question des unités s'était déjà posée. L'approximation SRM = Lovibond convenait à tout le monde, et surtout semblait pratiquée par les autres logiciels également. Mais c'est une question qui revient régulièrement sur les forums, et les réponses  ne me semblent pas complètement satisfaisantes.
Soit on accepte cette approximation, soit on est strict mais on casse la compatibilité avec une bonne partie des autres logiciels et tous ceux qui ont accepté cette équivalence...
